etc/docs/template/static/s/css/frontpage.css

```diff
@@ -0,0 +1,399 @@
+    border: 1px solid #3b2920;
+}
+
+.gsc-search-box.gsc-search-box-tools .gsc-search-box .gsc-input {
+    padding: 0;
+}
+
+div.gsc-input-box,
+.gsc-search-box .gsc-input>input:hover, .gsc-input-box-hover,
+.gsc-search-box .gsc-input>input:focus, .gsc-input-box-focus {
+    border: 0;
+    background: transparent;
+    box-shadow: none;
+}
+
+/* [name] selector is a hack to override google's stylsheet */
+.gsc-input input.gsc-input[name="search"] {
+    background-color: transparent !important;
+    color: white;
+    font-weight: 300;
+    font-size: 15px;
+    height: 1.3em !important;
+}
+
+.gsc-input input.gsc-input::-webkit-input-placeholder,
+.gsc-input input.gsc-input:-moz-input-placeholder,
+.gsc-input input.gsc-input::-moz-input-placeholder,
+.gsc-input input.gsc-input:-ms-input-placeholder {
+    color: #CCC;
+}
+
+/* Clear search button */
+.gsib_b {
+    display: none;
+}
+
+/* [title] selector is a hack to override google's stylesheet */
+input.gsc-search-button[title], input.gsc-search-button:hover[title], input.gsc-search-button:focus[title] {
+    border-color: transparent;
+    background-color: transparent;
+    padding: 0 8px;
+    box-sizing: content-box;
+    -moz-box-sizing: content-box;
+    -webkit-box-sizing: content-box;
+}
+
+#header-db .logo {
+    padding: 5px 0;
+}
+
+#header-db .search-db {
+    transition: width 0.3s linear;
+    -webkit-transition: width 0.3s linear;
+    -moz-transition: width 0.3s linear;
+    -o-transition: width 0.3s linear;
+    margin-left: 45px;
+    margin-top: 11px;
+    display: inline-block;
+}
+
+#header-db .search-db.narrow {
+    width: 98px;
+}
+
+#header-db .search-db.wide {
+    width: 226px;
+}
+
+/* Footer styles */
+.footer {
+    width:auto;
+    font-size:80%;
+    border:none;
+    padding: 20px 0;
+}
+
+.footer .copyright {
+    text-align: center;
+}
+
+.footer p {
+    margin: 1em 0;
+    padding: 0;
+    line-height:1.5;
+}
+
+.footer a {
+    color: #989898;
+    text-decoration: underline;
+}
+
+.footer h3 {
+    margin-top: 10px;
+    font-size: 16px;
+}
+
+.footer .box {
+    background-color: #f5f6f7;
+    min-height: 20px;
+    padding: 15px;
+    margin-bottom: 20px;
+    text-align: left;
+}
+
+.footer .box a {
+    color: #333;
+    text-decoration: none;
+}
+
+.footer .box a:hover {
+    text-decoration: underline;
+}
+
+.footer ul {
+    list-style-type: none;
+    padding-left: 20px;
+}
+
+.footer .section-0 {
+    padding-left: 100px;
+}
+
+.footer .section-1 {
+    padding-left: 46px;
+}
+
+.footer .section-2 {
+    padding-left: 64px;
+}
+
+.footer .section-3 {
+    padding-left: 49px;
+}
+
+
+/* Hero Styles */
+
+#hero {
+  background-color: #666;
+  color: #f3f4eb;
+  border-bottom: 1px solid #313030;
+  padding: 12px 0px;
+}
+
+#nodelogo {
+  background-image: url("../img/nodejs.png");
+  background-position: left top;
+  background-repeat: no-repeat;
+}
+
+#mongodbJVMlogo {
+  background-image: url("../img/mongoJVMlogo.png");
```

Comment:
Let's change this to: https://wiki.corp.mongodb.com/pages/viewpage.action?pageId=79305748&preview=/80089640/133270877/node.png :)

Reply:
Let's update this in my second PR - the `img` files don't exist in this PR since there are only changes to the site template (which in our case, doesn't include the images).